src/components/AppLayout/Sidebar/index.tsx

```diff
@@ -68,6 +68,8 @@ const lazyLoad = (path: string): React.ReactElement => {
   return wrapInSuspense(<Component />)
 }
 
+const isDesktop = process.env.REACT_APP_BUILD_FOR_DESKTOP
```

Comment:
You can reference `window.isDesktop` to get this too.

Reply:
Good to know.